xchange-coinsuper/src/main/java/org/knowm/xchange/coinsuper/service/CoinsuperTradeService.java

```diff
@@ -131,32 +132,31 @@ public OpenOrdersParams createOpenOrdersParams() {
     return new DefaultOpenOrdersParamCurrencyPair();
   }
 
-  @Override
-  public Collection<Order> getOrder(String... orderIds) throws IOException {
-
+  public Collection<Order> getOrderImpl(String... orderIds) throws IOException {
+    // This method possibly not working
     Collection<Order> orders = new ArrayList<>(orderIds.length);
 
-    Map<String, String> parameters = new HashMap<String, String>();
+    Map<String, String> parameters = new HashMap<>();
     StringBuilder builder = new StringBuilder();
-    int k = 1;
     for (String orderId : orderIds) {
       builder.append(orderId);
-      if (k > orderIds.length) builder.append(",");
+      builder.append(",");
```

Comment:
This control was an invalid statement. 

In this code, variable "k" is a constant which is set as 1(it is not updated). And (k > orderIds.length) is always false in the loop because when stack steps to this statement, there is at least one member in the orderIds list and "orderIds.length > 1".

Actually I should also delete "builder.append(",");", because it was not executed whole time. If you confirm, I can.

Reply:
Do you not want the comma? I think this should be an increasing k and check if this is the last object in the list shouldn't add the comma

Reply:
Actually, I do not know the intend of the original developer of the code, but I was just trying to clean the code with preserving the execution in the statement context.  I can just revert it back to the original. But the original code was not valid.

If it is desired to have the comma, I can do the loop accordingly. You tell me ? :)

Reply:
Let's keep the comma there and just fix the logic to stop right before the last one

Reply:
Okay

Reply:
Done